edenia/eden-spend-explorer
# style(webapp): styles refactor

### Styles refactor

### What does this PR do?

- Resolve #247 

### Steps to test

1. Run the project and open localhost:3000
2. see the improved styles

#### CheckList
